tasks/shared_libraries/README.md

```diff
@@ -0,0 +1,120 @@
+The following sections going into more detail on utilizing these libraries.
+
+- [Installing ORCA Shared Libraries](#installing-orca-shared-libraries) - Information on installing the libraries into your application
+- [Using ORCA Shared Libraries](#using-orca-shared-libraries) - Basic usage and documentation on the shared libraries.
+- [Testing ORCA Shared Libraries](#testing-orca-shared-libraries) - Information on testing the shared libraries package.
+- [Creating and Updating Shared Libraries](#creating-and-updating-shared-libraries) - Documentation on creating and updating shared libraries to use.
+
+
+## Installing ORCA Shared Libraries
+
+The ORCA shared libraries can be installed via `pip`. Users have the option of
+installing all shared libraries or only select libraries needed for their application.
+Since the package is not published on PyPI, users install the package by pointing
+to the `shared_libraries` directory. This can be done with relative paths. To
+install all shared libraries, run the following pip command.
+
+:::info
+
+The command below assumes you are running the install from within the task
+directory. Adjust the relative path accordingly based on where your requirements
+file or install script is located.
+
+:::
+
+```bash
+pip install ./shared_libraries/["all"]
+```
+
+If you only wish to install a specific library, denote that library during the
+install. An example of installing only the shared database library can be seen
+below.
+
+```bash
+pip install ./shared_libraries/["database"]
+```
+
+
+## Using ORCA Shared Libraries
+
+To utilize the shared libraries, user's can import items at the package or module
+level. To utilize all the functionality of a library import items at the package
+level like the following.
+
+```python
+from orca_shared import database
+```
+
+To utilize a specific module the user can import just the objects available in
+the module like the following.
+
+```python
+from orca_shared.database import shared_db
+```
+
+A complete listing of available packages and functions including usage is available
+[here](API.md).
+
+
+## Testing ORCA Shared Libraries
+
+To run unit tests for ORCA shared libraries run the `bin/run_tests.sh` script from the
+`tasks/shared_libraries` directory. Ideally, the tests should be run in a docker
+container. The following shows setting up a container to run the tests.
+
+```bash
+# Invoke a docker container in interactive mode.
+user$ docker run \
+      -it \
+      --rm \
+      -v /path/to/cumulus-orca/repo:/data \
+      amazonlinux:2 \
+      /bin/bash
+
+# Install the python development binaries
+bash-4.2# yum install python3-devel
+
+# In the container cd to /data
+bash-4.2# cd /data
+
+# Go to the task
+bash-4.2# cd tasks/shared_libraries
+
+# Run the tests
+bash-4.2# bin/run_tests.sh
+```
+
+Note that Bamboo will run this same script via the `bin/run_tests.sh` script found
+in the cumulus-orca base of the repo.
+
+
+## Creating and Updating Shared Libraries
```

Comment:
Add bumped version number

Reply:
Added bullet to specify updating the version to specify a change was made.